### ISSUE TITLE
listen only one song

### ISSUE THREAD

**Issue by HarryCover78** [librespot-org/librespot#392]
Good morning all :)

Thank you for all the work already done on your project.

I come to you because my librespot instalation works just for one song :(

(RPI 3B+ RASPIAN BUSTER LITE, connected at one LG bluetooth soundbar to bluealsa)

If I change the usual or if the song is finished and passes to the following, there is no sound ... :(

I can show you the result of any command to solve this problem, and I hope it will be useful to others :)

Yours truly.

PS: sorry for my english

**Comment by kaibenn** [comment]
I think I have the same problem, if I understand HarryCover78 correctly.

I am using librespot with the current librespot kodi addon (from awiouy) on a CoreElec installation.

Starting to playing a complete album I am used to start the first song and spotify automatically plays the complete album. This has worked good in the past. Since a while this often fails. Only the first song is in the playlist.

If I play local on the mobile, this works always.

As a workaround I can add the complete album to the playlist (spotify menu option on the album)

Best regards

**Comment by 5amSmith** [comment]
I have the same issue with LibreELEC on my Raspberry Pi.

**Comment by kaibenn** [comment]
I have not observed this problem anymore for a while, now. Was it related to the spotify side? And is it fixed, already? We will see.

**Comment by Malvineous** [comment]
I'm new to the code but I think the way it works is that some player (Spotify app, Kodi, etc.) tells Spotify what song to play, and Spotify passes that to librespot.  The player can pass either a song ID (of one song only) or a playlist ID.  If the playlist ID is sent, librespot will play all the songs on the playlist automatically, even if the player is closed.  But if the ID of only one song is passed through, then librespot will stop once that song is complete.

If you're only hearing one song, but it works with an official Spotify client (e.g. website), I think this is a problem with the player you're using rather than librespot.
